engines/director/lingo/lingo-builtins.cpp

```diff
@@ -19,6 +19,7 @@
  *
  */
 
+#include "common/str.h"
```

Comment:
Are you sure this is needed here? It is for sure included in tokenizer.h below

Reply:
Gah, code completion trying to be helpful. Removed